CONTRIBUTING.md

```diff
@@ -11,7 +11,7 @@ Thanks for taking the plunge!
   or related issues for context, before opening a new issue
 * Including minimal examples is greatly appreciated
 * If it's a bug, or unexpected behaviour, reproducing on the latest development version
-  (`Pkg.checkout("DataFrames")`) is a good gut check and can streamline the process,
+  (`Pkg.add(name="DataFrames", rev="master")`) is a good gut check and can streamline the process,
```

Comment:
Please remove this change - you mush have added it by accident from an urelated PR: https://github.com/JuliaData/DataFrames.jl/pull/2387